manifest/1.4.1.xml

```diff
@@ -0,0 +1,64 @@
+  <project groups="notdefault,sg-accel" name="go-metrics" path="godeps/src/github.com/rcrowley/go-metrics" remote="rcrowley" revision="7aeccdae5c4ea7140b90c8af1dcf9563065cc6dd" />
+  <project name="go-metrics" path="godeps/src/github.com/samuel/go-metrics" remote="samuel" revision="52e6232924c9e785c3c4117b63a3e58b1f724544" />
+  <project name="go-oidc" path="godeps/src/github.com/coreos/go-oidc" remote="coreos" revision="5aa9381f6e998aa16cc96b4347d33dcc29792864" />
+  <project name="go-systemd" path="godeps/src/github.com/coreos/go-systemd" remote="coreos" revision="1d9051fe7a349daf6dac904c0b277c3520c09368" />
+  <project name="go.assert" path="godeps/src/github.com/couchbaselabs/go.assert" remote="couchbaselabs" revision="cfb33e3a0dac05ae1ecbc0e97188c5cf746a1b78" />
+  <project name="gocb" path="godeps/src/github.com/couchbase/gocb" revision="67fb8fa4c62451372108f31c93034c07d9d9f4ab" />
+  <project name="gomemcached" path="godeps/src/github.com/couchbase/gomemcached" revision="6172a8c61c821c420071fe9e20e74d8e24c8cbd5" />
+  <project name="goutils" path="godeps/src/github.com/couchbase/goutils" revision="5823a0cbaaa9008406021dc5daf80125ea30bba6" />
+  <project name="lumberjack" path="godeps/src/github.com/natefinch/lumberjack" remote="natefinch" revision="dd45e6a67c53f673bb49ca8a001fd3a63ceb640e" />
+  <project name="mux" path="godeps/src/github.com/gorilla/mux" remote="gorilla" revision="9b36453141c35697401168b07f2c09fcff7721ce" />
+  <project name="net" path="godeps/src/golang.org/x/net" remote="couchbasedeps" revision="7006f7a1abbda37f3276216302f9eb7425e1dfa5" />
+  <project name="npipe" path="godeps/src/github.com/natefinch/npipe" remote="natefinch" revision="0938d701e50e580f5925c773055eb6d6b32a0cbc" />
+  <project name="osext" path="godeps/src/github.com/kardianos/osext" remote="kardianos" revision="29ae4ffbc9a6fe9fb2bc5029050ce6996ea1d3bc" />
+  <project name="otto" path="godeps/src/github.com/robertkrimen/otto" remote="couchbasedeps" revision="5282a5a45ba989692b3ae22f730fa6b9dd67662f" />
+  <project name="pkg" path="godeps/src/github.com/coreos/pkg" remote="coreos" revision="160ae6282d8c48a33d8c150e4e4817fdef8a5cde" />
+  <project name="retriever" path="godeps/src/github.com/couchbase/retriever" revision="19c5a5d92a2f34fb96ae91d26901e4a7076b8020" />
+  <project name="service" path="godeps/src/github.com/kardianos/service" remote="kardianos" revision="2954cfdd7b0c8ab45ef2aa22a44b5f086201836f" />
+  <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" revision="a22e64010d08574baa33332695abb27c8e50d251" />
+  <project name="sg-replicate" path="godeps/src/github.com/couchbaselabs/sg-replicate" remote="couchbaselabs" revision="24b8252856d71a541cbfeb2203cf7c0ce3634b2b" />
+  <project groups="notdefault,sg-accel" name="sync-gateway-accel" path="godeps/src/github.com/couchbaselabs/sync-gateway-accel" remote="couchbaselabs_private" revision="82be89009d2d9c0c3d546dd0f7d19428376497e7" upstream="master" />
```

Comment:
The second commit fixes it in manifest/1.4.0.xml, but I don't see it as fixed in manifest/1.4.1.xml